### DISCUSSION TITLE
How to apply ssl to Gitea

### DISCUSSION THREAD

**Discussion by shripadaier** [community/community#134282] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to apply ssl to Gitea.
We have a Gitea Server running on the Windows platform. We have renewed our SSL certificate from GoDaddy and need information on how to apply it to Gitea.

**Comment by queenofcorgis** [comment]
Hi @shripadaier ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Gitea](https://about.gitea.com/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
